tests/test_marathon_tools.py

```diff
@@ -241,28 +241,28 @@ def test_read_service_config_and_deployments(self):
 
     def test_load_marathon_config(self):
         expected = {'foo': 'bar'}
+        from_file = {'marathon_config': {'foo': 'bar'}}
         file_mock = mock.MagicMock(spec=file)
         with contextlib.nested(
-            mock.patch('paasta_tools.marathon_tools.open', create=True, return_value=file_mock),
-            mock.patch('json.load', autospec=True, return_value=expected)
+            mock.patch('paasta_tools.utils.open', create=True, return_value=file_mock),
```

Comment:
To be clear, are you suggesting removing `test_load_marathon_config` and `test_load_marathon_config_path_dne`, and similar with chronos? I just want to make sure I don't remove the wrong tests

And should I add something to test_utils to make up for it?